docs/decisions/0003-reduce-bokchoy-testing.rst

```diff
@@ -0,0 +1,58 @@
+Status
+~~~~~~
+Draft
+
+
+Context
+~~~~~~~
+
+edx-platform bokchoy tests are slow, flaky and difficult to debug.  A quick assessment of their value shows that they might be more trouble than they are worth.  And that we might get the same benefit with far fewer tests.
+
+Baseline Data:(Last 7 days)
+---------------------------
+
+    Total number of builds: 253(across 106 PRs)
+
+    Failures: 49(across 24 PRs)
+        True Failures: 10(across 6 PRs)
+        Failures that wouldn’t be caught by other test: 3(on 1 PR)
+
+Color
+=====
+
+    Of the real failures found, there was one PR which had a failure that was only found via bokchoy and a115 tests.
+        - This PR made a JS change which would have broken many pages from loading.
```

Comment:
Remove indent, and use `*`.  See https://github.com/edx/mdrst/blob/master/mdrst.rst#lists.